src/ActionModal/components/ActionModal.vue

```diff
@@ -300,12 +330,32 @@ export default {
           }
         ]
       }
+      if (this.session.sessionType === `extension`) {
+        return [
+          {
+            key: `Lunie Chrome Extension`,
+            value: signWithExtension
+          }
+        ]
+      }
       return [
         {
           key: `(Unsafe) Local Account`,
           value: signWithLocalKeystore
         }
       ]
+    },
+    submitButtonCaption() {
+      return (() => {
```

Comment:
I need to know your thinking behind this `function x{return () => {}()}` call?

:)

Reply:
brainfreez ;D need to remove it